nnfabrik/main.py

```diff
@@ -351,3 +351,77 @@ class Seed(dj.Manual):
     definition = """
     seed:   int     # Random seed that is passed to the model- and dataset-builder
     """
+
+
+def custom_nnfabrik(
+    schema,
+    use_common_fabrikant=True,
+    use_common_seed=False,
+    module_name=None,
+    context=None,
+    spawn_existing_tables=False,
+):
+    """
```

Comment:
Can you add a code example of how to use it? Something that shows the difference between old and new usage.

Reply:
Great point - added `Examples` to the docstring.